### ISSUE TITLE
Numpy error causes surface module to fail

### ISSUE THREAD

**Issue by araikes** [reuter-lab/qatools-python#64]
## Description

I have a singularity image of `fsqc` that I built today which uses the current state of this repository.  For all participants, surface plotting fails due to a numpy 2.0 change from `np.mat` to `np.asmatrix`.

## Steps to Reproduce

`singularity exec /groups/adamraikes/singularity_images/fsqc-2.1.0.sif /app/fsqc/run_fsqc --subjects_dir $PWD/fastsurfer --output_dir $PWD/fsqc --shape --screenshots-html --surfaces-html --skullstrip-html --fornix-html --hypothalamus-html --outlier --fastsurfer`

```
-----------------------------
Creating surface plots ...

[ERROR: fsqcMain.py: 2219]: ERROR: surfaces module failed for subject sub-005_ses-02
[ERROR: fsqcMain.py: 2222]: Reason: `np.mat` was removed in the NumPy 2.0 release. Use `np.asmatrix` instead.
-----------------------------
```

### Execution

 Singularity definition file:
 ```
BootStrap: docker
From: ubuntu:22.04

%post
    # Update OS
    apt-get update && apt-get install -y --no-install-recommends \
        gcc \
        git \
        time \
        zlib1g-dev \
        libjpeg-dev \
        python3 \
        python3-dev \
        python3-pip \
        python3-setuptools \
        python3-wheel

    # Clone fsqc repository
    git clone https://github.com/Deep-MI/fsqc.git /app/fsqc

    # Install Python packages from requirements file
    pip3 install -r /app/fsqc/requirements.txt

    # Set environment variables
    export OMP_NUM_THREADS=1
    export MPLCONFIGDIR=/tmp/mplconfigdir

%environment
    # Define environment variables for runtime
    export OMP_NUM_THREADS=1
    export MPLCONFIGDIR=/tmp/mplconfigdir

%runscript
    # This is the default action when running the container
    python3 /app/fsqc/run_fsqc
```

Run Command:

**Comment by kdiers** [comment]
Hello, thanks for your report. We will likely issue a fix shortly.

**Comment by kdiers** [comment]
This issue has now been fixed in the dev branch; we will create a new release (2.1.1) and new docker images during the next days.

**Comment by kdiers** [comment]
Closing this issue, was addressed in release [v2.1.1](https://github.com/Deep-MI/fsqc/releases/tag/v2.1.1).
